pip_custom_platform/pymonkey.py

```diff
@@ -0,0 +1,45 @@
+def pymonkey_argparse(argv):
+    def _add_platform_param(parser):
+        parser.add_argument(
+            '--platform',
+            help='Custom platform name (default is auto-detected)',
```

Comment:
yeah I can probably make a lazy object of some sort that displays.   The problem is I can't import distutils.util during argument parsing or the whole thing falls apart.  I'll make something in a followup branch